app/models/identity.rb

```diff
@@ -6,15 +6,22 @@ class Identity < ApplicationRecord
 
   delegate :metadata, to: :sp, prefix: true
 
+  # rubocop:disable Rails/InverseOf
+  belongs_to :service_provider_record,
```

Comment:
I *really* wanted to call this `service_provider_record` but `alias_attribute` is not powerful enough to let us do this cleanly.... got a bunch of stack to deep errors.

Maybe we can rename the column to `issuer` ... someday...

Reply:
I agree we should rename this issuer ...someday